src/client/errors.ts

```diff
@@ -2,32 +2,89 @@
  * @module Errors
  */
 
+/**
+  * Error codes returned by actions involving creating applications.
+  */
 export enum OpenError {
+    /**
+     * Indicates that an application is unable to be found in the AppDir.
+     */
     AppNotFound = 'AppNotFound',
+    /**
+     * Indicates that an application cannot be started from an OF manifest.
+     */
     ErrorOnLaunch = 'ErrorOnLaunch',
+    /**
+     * Indicates that an application was not created in a timely manner, and the provider timed out.
+     */
     AppTimeout = 'AppTimeout',
+    /**
+     * Currently unused.
+     */
     ResolverUnavailable = 'ResolverUnavailable',
```

Comment:
I think get rid of, or indicate what we expect this to be used for

Reply:
I presume this was written for future use; so I am not sure. Maybe I will resolve merge clashes and see if it's still there. Either way I am not sure I want to do that in this doc PR